WordPress/WordPressTest/Notification/Controllers/NotificationsViewControllerTests.swift

```diff
@@ -52,6 +52,13 @@ final class NotificationsViewControllerTests: XCTestCase {
         postAccountChangeNotification()
 
         // Then
+        // TODO: rework this unit test
+        let expectation = self.expectation(description: "setBadgeCount")
+        DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(100)) {
```

Comment:
These tests are failing because the deprecated `applicationIconBadgeNumber` property now works asynchronously. It's not great, but we can rework it when we remove the deprecated API.